buildSrc/src/main/java/org/elasticsearch/gradle/BwcVersions.java

```diff
@@ -345,7 +349,14 @@ public void compareToAuthoritative(List<Version> authoritativeReleasedVersions)
         wireCompat.addAll(groupByMajor.get(currentVersion.getMajor()));
         wireCompat.sort(Version::compareTo);
 
-        return unmodifiableList(wireCompat);
+        return Architecture.current() == Architecture.AARCH64
+            ? unmodifiableList(filterAarch64SupportedVersions(wireCompat))
+            : unmodifiableList(wireCompat);
+    }
+
+    @NotNull
+    private List<Version> filterAarch64SupportedVersions(List<Version> wireCompat) {
```

Comment:
What do you think about calling this `filterSupportedVersions` and moving the architecture check here? Then there's only only place where we're checking whether we need to filter.

Reply:
👍